Fancyauth/Steam/SteamEventForwarder.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Linq;
+using System.Data.Entity;
+using System.Threading.Tasks;
+using SteamKit2;
+using Fancyauth.Steam;
+
+namespace Fancyauth.Steam
+{
+    public class SteamEventForwarder : ISteamEventForwarder
```

Comment:
Why is this not a normal Builtin Plugin? Seems better than a new interface with implementation IMO. Especially because it is nothing more than a builtin plugin.